weblab/core/tests/test_combine.py

```diff
@@ -87,7 +87,8 @@ def test_xml_doc(self, writer, manifest):
         writer.add_file('test.txt', fmt='txt')
 
         doc = ET.tostring(writer.xml_doc.getroot()).decode()
-        assert doc == manifest
+        manifest_str = ET.tostring(ET.fromstring(manifest)).decode()
+        assert doc == manifest_str is not None
```

Comment:
This seems a very odd test! A comment explaining why would be helpful!

Reply:
It's because the order of properties is different in python 3.8 and I'm trying to find a way that can deal with that. This seems not to be working though.

Reply:
Oh fun :(
I'd stick to testing on just 3.5-3.7 for now, and open another issue for supporting 3.8 & 3.9. We control deployments so can control what version is used much more easily.